Formula/grpc.rb

```diff
@@ -80,5 +80,6 @@ def install
     EOS
     system ENV.cc, "test.cpp", "-I#{include}", "-L#{lib}", "-lgrpc", "-o", "test"
     system "./test"
+    assert_match "Received an error when querying services endpoint." shell_output("grpc_cli ls localhost:#{free_port}", 1)
```

Comment:
Forgot a comma, sorry
```suggestion
    assert_match "Received an error when querying services endpoint.", shell_output("grpc_cli ls localhost:#{free_port}", 1)
```

Reply:
We also need to redirect stderr to stdout, I will update the PR.